rectools/visuals/metrics_app.py

```diff
@@ -0,0 +1,120 @@
+import typing as tp
+
+import ipywidgets as widgets
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objs as go
+from IPython.display import display
+
+from rectools import Columns
+
+DEFAULT_WIDTH = 800
+DEFAULT_HEIGHT = 600
+
+
+class MetricsApp:
+    """
+    Jupyter widgets app for metric visualization and comparison.
+    Do not create instances of this class directly. Use `MetricsApp.construct` method instead.
+    """
+
+    def __init__(
+        self,
+        df_metrics_data: pd.DataFrame,
+        show_legend: bool = True,
+        auto_display: bool = True,
+        plotly_kwargs: tp.Optional[tp.Dict[str, tp.Any]] = None,
+    ):
+        self.metrics_data = df_metrics_data
+        self.show_legend = show_legend
+        self.auto_display = auto_display
+        self.default_plotly_kwargs = {"width": DEFAULT_WIDTH, "height": DEFAULT_HEIGHT}
```

Comment:
I added this default option because of chart trembling after first interaction with any widget.
I agree that this approach seems dirty, so I've update code, now this width and height kwargs directly used only in _update_chart method.

Reply:
Probably outdated, because I removed `self.default_plotly_kwargs` from current version